src/pages/experiments/[experimentId]/plots-and-tables/frequency/components/SelectCellSets.jsx

```diff
@@ -3,31 +3,48 @@ import PropTypes from 'prop-types';
 import {
   Form,
   Select,
+  Tooltip,
 } from 'antd';
 
 const SelectCellSets = (props) => {
   const { onUpdate, config, cellSets } = props;
-
   const changeClusters = (val) => {
-    onUpdate({ chosenClusters: val.key });
+    const newValue = val.key.toLowerCase();
+    onUpdate({ chosenClusters: newValue });
   };
-
+  let menuValue = config.metadata;
+  let disabled = false;
+  let toolTipText;
   const changeMetadata = (val) => {
-    onUpdate({ metadata: val.key });
+    const newValue = val.key.toLowerCase();
+    onUpdate({ metadata: newValue });
   };
 
   const generateCellOptions = (type) => {
     if (cellSets.loading) {
       return [];
     }
     const options = cellSets.hierarchy.map(({ key }) => ({ value: key }));
-
     const filteredOptions = options.filter((element) => (
       cellSets.properties[element.value].type === type
     ));
-    return filteredOptions;
+    if (!filteredOptions.length) {
+      return [];
+    }
+    // making the options with capital letters as per requirement
+    const upperCaseOptions = [];
+    filteredOptions.forEach((option) => {
+      upperCaseOptions.push({
+        value: option.value.charAt(0).toUpperCase() + option.value.slice(1),
```

Comment:
I see this operation repeated several times in this file (lines : 38, 59, 77) : `option.value.charAt(0).toUpperCase() + option.value.slice(1)`. Can you refactor it into a function?